defaults.go

```diff
@@ -45,16 +45,51 @@ func WithBlocks(blocks []slack.Block) ReplyOption {
 	}
 }
 
+// ThreadResponse specifies the reply to be inside a thread of the original message
+func ThreadResponse() ReplyOption {
+	return func(defaults *ReplyDefaults) {
+		defaults.ThreadResponse = true
+	}
+}
```

Comment:
I would suggest renaming this to “WithThreadReply” and taking a Boolean as parameter.

1) To be consistent with the other options, they all start with “With”
2) Just in case a user wants to be explicit about wanting a non thread reply by passing false.

Reply:
Nicely put. Done